UI/update/win-update.cpp

```diff
@@ -50,30 +50,13 @@ using namespace updater;
 
 /* ------------------------------------------------------------------------ */
 
-#if defined(OBS_RELEASE_CANDIDATE) && OBS_RELEASE_CANDIDATE > 0
-#define CUR_VER                                                               \
-	((uint64_t)OBS_RELEASE_CANDIDATE_VER << 16ULL | OBS_RELEASE_CANDIDATE \
-								<< 8ULL)
-#define PRE_RELEASE true
-#elif OBS_BETA > 0
-#define CUR_VER ((uint64_t)OBS_BETA_VER << 16ULL | OBS_BETA)
-#define PRE_RELEASE true
-#elif defined(OBS_COMMIT)
-#define CUR_VER 1 << 16ULL
-#define CUR_COMMIT OBS_COMMIT
-#define PRE_RELEASE true
-#else
-#define CUR_VER ((uint64_t)LIBOBS_API_VER << 16ULL)
-#define PRE_RELEASE false
-#endif
-
-#ifndef CUR_COMMIT
-#define CUR_COMMIT "00000000"
-#endif
+constexpr uint64_t kCurVer = (uint64_t)LIBOBS_API_VER << 16ULL |
```

Comment:
It's not 1976 anymore, _if_ we refactor this, could we please use readable variable names (i.e. `kCurrentVersion`).

I'm not sure what `k` in Hungarian Notation means, but when it comes to the style it might be one of the very few things I agree with Linus Torvalds on. 😬

Reply:
`k` is a common prefix for global constants in various C++ projects, I think most just adapted that from the Google style guide, though it may predate that. Since we don't have a C++ style guide I just started using it in my C++ code to keep things clearer.

Reply:
On second thought, these don't need to be globals though, I just replaced the macros very literally. I just moved this to the function scope.

Reply:
I've seen a description that states:
> k - a constant or immutable variable

Globals are usually prefixed with `g` or `g_`.